Peekabook/Peekabook/Presentation/AddBook/VC/BookSearchVC.swift

```diff
@@ -70,6 +85,21 @@ extension BookSearchVC {
         headerTitle.snp.makeConstraints { make in
             make.center.equalToSuperview()
         }
+        
+        searchField.snp.makeConstraints { make in
+            make.top.equalTo(headerView.snp.bottom).offset(18)
+            make.leading.equalToSuperview().offset(20)
+            make.trailing.equalToSuperview().inset(20)
+            make.centerX.equalToSuperview()
```

Comment:
```suggestion
            make.leading.trailing.equalToSuperview().inset(20)
```
이건 어떨까용...? 개인적인 의견입니돠